javaparser-core/src/main/java/com/github/javaparser/ast/type/IntersectionType.java

```diff
@@ -108,6 +108,11 @@ public IntersectionType setAnnotations(NodeList<AnnotationExpr> annotations) {
         return (IntersectionType) super.setAnnotations(annotations);
     }
 
+    @Override
+    public String toDescriptor() {
+        return null;
```

Comment:
The expected value when calling this method is a string. I think it's better to return an empty string because if we return a null value it may throw exceptions for users.